### ISSUE TITLE
No append for dict

### ISSUE THREAD

**Issue by eanon** [jcontini/facebook-scraper#12]
Also, in addition to issue #11, I got error "AttributeError: 'dict' object has no attribute 'append'" on
```profiles.append(d)``` (line 253 in parse_profiles).

[Python 3.7, Windows 7]

**Comment by eanon** [comment]
At first, I changed profiles.append by profiles.update, but it was a wrong idea... The issue is that profiles should be a list, while it was a dictionnary reading an empty or missing profiles.json. Then, I simply forced the type of profiles to list when it appears to be an empty dict.

It was:
```    with open(db_profiles) as f:
        profiles = json.load(f)
```

And it's now:
```    with open(db_profiles) as f:
        profiles = json.load(f)
        if not bool(profiles):
            profiles = []
```

It works like this on my part! I let you test and close if it's OK for you too...
